Expecto.Differ/Library.fs

```diff
@@ -0,0 +1,45 @@
+module Expecto.Differ
+open System
+open DiffPlex.DiffBuilder
+open DiffPlex.DiffBuilder.Model
+
+let colorizedDiff (colorizeText: ConsoleColor -> string -> string) first second =
+  let first, second =
+    match box first, box second with
+    | (:? string as f), (:? string as s) ->
+      string f, string s
+    | f, s ->
+      sprintf "%A" f, sprintf "%A" s
+
+  let differ = SideBySideDiffBuilder(DiffPlex.Differ())
+  let diff = differ.BuildDiffModel(first, second)
+
+  let coloredText typ text =
+    match typ with
+    | ChangeType.Inserted -> colorizeText ConsoleColor.Green text
+    | ChangeType.Deleted -> colorizeText ConsoleColor.Red text
+    | ChangeType.Modified -> colorizeText ConsoleColor.Blue text
+    | ChangeType.Imaginary -> colorizeText ConsoleColor.Yellow text
+    | ChangeType.Unchanged | ChangeType.Imaginary | _ -> text
+
+  let colorizedDiff (lines: DiffPiece seq) =
+    lines
+    |> Seq.map (fun line ->
```

Comment:
split callback to own function, to avoid some indentation

Reply:
Yea, I can experiment. I actually barely refactored yet at all – I just wanted to get things up and working quickly to see if it's worth finishing or not.